modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+
+[id="installation-osp-deploying-bare-metal-machines_{context}"]
+= Deploying a cluster with bare metal machines
+
+If you want your cluster to use bare-metal machines, modify the
+ifdef::osp-ipi[`install-config.yaml`]
+ifndef::osp-ipi[`inventory.yaml`]
+file. Your cluster can have both control plane and compute machines running on bare metal, or just compute machines.
+

```

Comment:
When following the OpenStack guide on Ironic enabled overcloud and then following this guide the installation will fail since the provider network is unable to work with floating IPs.
I think that adding a note to our customers here is important so they will be able to take into consideration what type of deployment they need - FIP or FIPless

[NOTE]
====
Make sure that you adjust your install-config.yaml to support FIP or FIPless OpenShift deployment based on the openstack network provided for baremetal installation.
====

Reply:
Thanks!